src/components/tabs/_tabs.scss

```diff
@@ -1,6 +1,35 @@
 .cmp-tabs {}
 .cmp-tabs__tablist {}
-.cmp-tabs__tab {}
-.cmp-tabs__tab--active {}
+.cmp-tabs__tab {
+    height: 68px;
+    width: 260px;
```

Comment:
May need to define this more responsively. But we can also do that in a second pass.